notebooks/Add GPS Metadata Fields.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.                    if not set(set([0, 1, 2, 4, 5])).issubset(values):
</pre><p>I don't think you need to create a set of set, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>